src/components/config/external/chatgpt/Config.vue

```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable prettier/prettier -->
```

Comment:
Remove this eslint-disable, the file should be respecting prettier configurations